python/tk_desktop2/websockets/shotgun_cert_handler.py

```diff
@@ -84,6 +84,12 @@ def _write_cert(self, path, cert):
         :param str path: Path to file that should be written.
         :param str cert: Certificate taken from Shotgun.
         """
+        if len(cert) < 5:
```

Comment:
Quick comment here to explain the minimum length of 5? I assume it's the begin/end tag lines plus 3 lines of cert data, but I don't know why that's the minimum viable cert.

Reply:
yep, good idea, will do!